packages/components/src/accordion/docs/Accordion.stories.mdx

```diff
@@ -21,15 +21,16 @@ import { InnerItem, Item } from "@components/collection";
     githubPath="/packages/components/src/accordion/src"
 />
 
-## Accessibility
+## Guidelines
 
-The accordion header that shows and hides the content must be an [heading element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
+### Accessibility
 
-That heading element should have a level that is appropriate for the information architecture of the page. You can choose the appropriate level by selecting one of Orbit heading element accordingly.
+- The accordion header that shows and hides the content must be an [heading element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
+- The heading element should have a level that is appropriate for the information architecture of the page. You can choose the appropriate level by selecting one of Orbit [Heading](?path=/docs/heading--default-story) element accordingly.
 
 ## Usage
 
-We do recommend that you specify a unique `key` prop for every `<Item>`. If you choose to omit the `key` prop, a key matching the `<Item>` position will be generated. For example, `"0"` would be the generated key for the first `<Item>` of an `<Accordion>` component without keys.
+> We do recommend that you specify a unique `key` prop for every `<Item>`. If you choose to omit the `key` prop, a key matching the `<Item>` position will be generated. For example, `"0"` would be the generated key for the first `<Item>` of an `<Accordion>` component without keys.
```

Comment:
If we want to put the emphasis on Accessibility within Orbit, we should avoid these Accessibility failures.
Markdown is not the best for styling content as it has limited option to highlight or indent content, but that solution (`>`) is a `blockquote`.
And the usage is a [failure of SC 1.3.1](https://www.w3.org/TR/2007/WD-WCAG20-TECHS-20070517/Overview.html#G115)
> Failure Example 3: Using blockquote elements to provide additional indentation

Blockquotes should only be used to refer to sourced content

Would it be ok to leave it as a plain text?
(This applies to other usage in the PR)

Reply:
You're right.

What do you suggest as an alternative to highlight a particular piece of content?

Reply:
It depends, the idea is to highlight the important stuff the user should see first. Without any specific I am not sure the result would be the same.

Furthermore, the solution you proposed might works for components pages but not for the other documentation pages like "Style props", or "installation"